src/main/webapp/issueList/IssueList.js

```diff
@@ -0,0 +1,222 @@
+import { Component } from 'react';
+import { hh, div, h1, h } from 'react-hyperscript-helpers';
+import { TableComponent } from "../components/TableComponent";
+import { Project } from '../util/ajax';
+import { Spinner } from "../components/Spinner";
+import { spinnerService } from "../util/spinner-service";
+import { formatDataPrintableFormat } from "../util/TableUtil";
+import { printData } from "../util/Utils";
+import { Link } from 'react-router-dom';
+import { format } from 'date-fns';
+import isNil from 'lodash/isNil';
+import '../index.css';
+
+const stylesHeader = {
+  pageTitle: {
+    fontWeight: '700', margin: '20px 0', fontSize: '35px', display: 'block'
+  }
+};
+
+const SPINNER_NAME = 'ISSUE_LIST';
+const styles = {
+  project: {
+    projectKeyWidth: '140px',
+    projectWidth: '750px',
+    titleWidth: '280px',
+    typeWidth: '193px'
+  }
+}
+
+const SIZE_PER_PAGE_LIST = [
+  { text: '10', value: 10 },
+  { text: '25', value: 25 },
+  { text: '50', value: 50 },
+  { text: '100', value: 100 }];
+
+const columns = [
+  {
+    dataField: 'id',
+    text: 'Id',
+    hidden: true,
+    csvExport: false
+  }, {
+    dataField: 'projectKey',
+    text: 'Project',
+    sort: true,
+    headerStyle: (column, colIndex) => {
+      return { width: styles.project.projectKeyWidth };
+    },
+    formatter: (cell, row, rowIndex, colIndex) =>
```

Comment:
This links can be moved to a function like this 
https://github.com/broadinstitute/orsp-pub/pull/252/files#diff-a9f4747606a7ea1e1584bf172822bf8fR50
```suggestion
        formatter: (cell, row, rowIndex, colIndex) => {
    return div({}, [
      linkFormatter(row, row.title)
    ])
  }
```